manifest.json

```diff
@@ -23,17 +23,23 @@
     "properties": {
       "model": {
         "default": "o1-mini",
+        "description": "The LLM model you wish to use",
+        "examples": ["openai/gpt-4o", "openai/o1-mini"],
         "type": "string"
       },
       "openAiBaseUrl": {
+        "description": "The base URL for the OpenAI API",
+        "examples": ["https://openrouter.ai/api/v1", "https://api.openai.com/v1"],
         "type": "string"
       },
       "similarityThreshold": {
         "default": 0.9,
+        "description": "When fetching embeddings context, the similarity threshold to use (1- similarityThreshold)",
         "type": "number"
```

Comment:
Do we want some examples for similarityThreshold and maxTokens? Or do you use that only for the values that have no default?

Reply:
My thoughts are: 

1. If it's obvious and the `input` is filled with the value type (number in this case) then it's pointless and would look silly
2. If the value is simple and displayed within the description then again no need for a example

These are planned to be displayed above/below the input areas like 

`When fetching embeddings context...`
`Examples: [<examples in this array>]`

Something like that but within the tooltip. So the user would see how to write this prop 3 different times if we also included `examples` because it exists in the description and `input` ele.